cineast-runtime/src/main/java/org/vitrivr/cineast/standalone/run/GenericExtractionItemHandler.java

```diff
@@ -383,15 +388,17 @@ private void shutdown() {
       }
 
       /* Get the appropriate handler for this item. We ignore image sequences if they're not specified because they support the same file types as images*/
-      if (handlerCache.entrySet().stream().filter(handler -> handler != null && handler.getKey() != MediaType.IMAGE_SEQUENCE)
-          .anyMatch(handler -> handler.getValue() != null && handler.getValue().getKey() != null && handler.getValue().getKey().supportedFiles().contains(type))) {
-        return new ImmutablePair<>(item, handlerCache.entrySet().stream()
-            .filter(handler -> handler != null && handler.getValue().getKey() != null && handler.getValue().getKey().supportedFiles().contains(type) && handler.getKey() != MediaType.IMAGE_SEQUENCE).findFirst()
-            .get().getKey());
+      Optional<Map.Entry<MediaType, Pair<Decoder, Segmenter>>> match = handlerCache.entrySet().stream().filter(handler -> handler != null && handler.getValue().getKey() != null && handler.getValue().getKey().supportedFiles().contains(type) && handler.getKey() != MediaType.IMAGE_SEQUENCE).findFirst();
+      if (match.isPresent()) {
+        return new ImmutablePair<>(item, match.get().getKey());
       } else {
-        /* If no appropriate handler is found, we log an error and continue with the next item */
-        LOGGER.error("No matching handlers found for type {} and item {}", type, item);
-        handlers.forEach((key, value) -> LOGGER.debug(key + " | " + value));
+        if (Files.isDirectory(item.getPathForExtraction())) {
+          LOGGER.debug("Not processing directory {}. Its content might be processed, but the directory itself is only processed for IMAGE_SEQUENCES. If you wish to use those, please specify the type explicitly in the extraction config.", item.getPathForExtraction());
```

Comment:
This addresses the comment of @lucaro . Skipping directories was implemented because there was confusion about why folders were logged with an error-message